code/parse/sexp.cpp

```diff
@@ -10145,9 +10145,11 @@ int test_argument_nodes_for_condition(int n, int condition_node, int *num_true,
 	return num_valid_arguments;
 }
 
+enum class STRDUP_STATUS { ALREADY_DUPPED, DUP_NEEDED, DUP_NOT_NEEDED };
```

Comment:
Please make a brief doxy explaining what the enum class is for.  The enumerations appear to be aptly named, so I don't believe they require to be individually documented.

I do recommend having one enumeration per line, however.

Reply:
I view this as an extension of the following function definition, so I don't think it requires a multi-line declaration or a doxy.  I've added a doxy anyway, but I left it single-line so that it's visually close to the function.

Reply:
Yep, that's what I was suggesting, a one-liner doxy. 👍

Reply:
Well... I'll allow it.